bin/improver-combine

```diff
@@ -46,28 +46,29 @@ def main():
     """Load in arguments for the cube combiner plugin.
     """
     parser = ArgParser(
-        description="Combine the input cubes into a single cube using "
+        description="Combine the input files into a single file using "
                     "the requested operation e.g. + - min max etc.")
     parser.add_argument("input_filenames", metavar="INPUT_FILENAMES",
                         nargs="+", type=str,
-                        help="A path to an input NETCDF files."
-                        " One cube per NetCDF file. The resulting"
-                        " cube will be based on the first cube but"
+                        help="Paths to the input NETCDF files. Each input"
```

Comment:
(If you happen to edit this file again - the case of NetCDF should probably be changed).